src/main/java/jenkins/security/plugins/ldap/LDAPConfiguration.java

```diff
@@ -573,8 +599,9 @@ public ApplicationContext createApplicationContext(LDAPSecurityRealm realm) {
         }
         contextSource.setReferral("follow");
         Map<String, Object> vars = new HashMap<>();
-        vars.put("com.sun.jndi.ldap.connect.timeout", "30000"); // timeout if no connection after 30 seconds
-        vars.put("com.sun.jndi.ldap.read.timeout", "60000"); // timeout if no response after 60 seconds
+        vars.put("com.sun.jndi.ldap.connect.pool", "true");
```

Comment:
Are we sure we want this, especially given it is not configurable?

Reply:
https://docs.spring.io/spring-ldap/docs/1.3.2.RELEASE/reference/html/pooling.html. Implies this poolingbis bad and there is no validation of the context.

Reply:
the idea was to have the exact same code/configuration for this test and how connection are created when used for authz see https://github.com/jenkinsci/ldap-plugin/blob/992524769cc9616d69b035b247d21badd9baafc1/src/main/java/jenkins/security/plugins/ldap/LDAPConfiguration.java#L602
it was totally different which is not better

Reply:
AHH, that may explain some random hangs. Of. The plugin then that are fixed by restarting.
I suggest pooling the s disabled everywhere, or isboptbin

Reply:
well sounds good. disabled per default but a configurable option?

Reply:
well there is still an option to add env props